contracts/smart-account/modules/PasskeyValidationModules/Secp256r1.sol

```diff
@@ -158,7 +158,7 @@ library Secp256r1 {
         return ret;
     }
 
-    // Wrapper for built-in BigNumber_modexp (contract 0x5) as described here. https://github.com/ethereum/EIPs/pull/198
+    // WraPPer for built-in BigNumber_modexp (contract 0x5) as described here. https://github.com/ethereum/EIPs/pull/198
```

Comment:
what is the significance of the `PP` in the comment?

Reply:
Why solhint is giving warning for small pp? Wrapper with small p seems to be the correct word